backend/CircleForms/appsettings.Development.json

```diff
@@ -1,15 +1,23 @@
 {
   "Logging": {
-    "LogLevel": {
+    "MinimumLevel": {
```

Comment:
Doesn't that should be overwritten by .Development config, and "Logging" already represents minimal level, isn't it

Reply:
It does not, at least not in Serilog